enrollment-server-onboarding/src/main/java/com/wultra/app/onboardingserver/statemachine/service/StateMachineService.java

```diff
@@ -123,6 +134,33 @@ public Message<OnboardingEvent> createMessage(OwnerId ownerId, String processId,
                 .build();
     }
 
+    /**
+     *
+     */
+    @Transactional(readOnly = true)
+    public void changeMachineStatesInBatch() {
+        try (Stream<IdentityVerificationEntity> stream = identityVerificationService.streamAllIdentityVerificationsToChangeState().parallel()) {
+            stream.forEach(this::changeMachineState);
```

Comment:
It would be nice to add the number of processed identity verifications per batch, probably in `debug` level.